formats/html.ts

```diff
@@ -21,24 +73,340 @@ export class HtmlImporter extends FormatImporter {
 			return;
 		}
 
-		let results: ImportResult = {
-			total: 0,
+		const results = await Promise.all(files
+			.map(file => this.processFile(folder, file)));
+		this.showResult({
+			total: results.map(({ total }) => total).reduce((left, right) => left + right, 0),
+			failed: results.map(({ failed }) => failed).flat(),
+			skipped: results.map(({ skipped }) => skipped).flat()
+		});
+	}
+
+	async processFile(folder: TFolder, file: PickedFile) {
+		const results: ImportResult = {
+			total: 1,
+			failed: [],
 			skipped: [],
-			failed: []
 		};
+		if (!(file instanceof NodePickedFile)) {
```

Comment:
Fixed. Now it will only not import local files on mobile. It seems like you cannot import local files because you cannot read an arbitrary filepath using browser APIs only.

Well, except that you can actually use Capacitor plugins in Obsidian plugins as long as the corresponding native Capacitor plugin is included in the official Obsidian build, like `@capacitor/filesystem`: https://github.com/polyipseity/obsidian-plugin-library/blob/a8caa2e3fb7bc45883c876f86413b1b52fd93c6d/sources/obsidian.ts#L491-L496. So this might be a way to provide attachment support on mobile.